resqpy/multiprocessing/multiprocessing.py

```diff
@@ -56,44 +56,45 @@ def function_multiprocessing(
             used when calling the function.
         recombined_epc (Path/str): A pathlib Path or path string of
             where the combined epc will be saved.
-        cluster (LocalCluster/JobQueueCluster): a LocalCluster is a Dask cluster on a
+        cluster: if using the Dask backend, a LocalCluster is a Dask cluster on a
             local machine. If using a job queing system, a JobQueueCluster can be used
             such as an SGECluster, SLURMCluster, PBSCluster, LSFCluster etc.
         consolidate (bool): if True and an equivalent part already exists in
             a model, it is not duplicated and the uuids are noted as equivalent.
         require_success (bool): if True and any instance fails, then an exception is
             raised
+        tmp_dir_path (str): path where the temporary directory is saved. Defaults to
+            the calling code directory.
+        backend (str): the joblib parallel backend used. Dask is used by default
+            so a Dask cluster must be passed to the cluster argument.
 
     Returns:
         success_list (List[bool]): A boolean list of successful function calls.
 
     Note:
-        This function uses the Dask backend to run the given function in parallel, so a
-        Dask cluster must be setup and passed as an argument. Dask will need to be
-        installed in the Python environment because it is not a dependency of the
-        project. More info can be found at 
+        This function uses the Dask backend by default to run the given function in
+        parallel, so a Dask cluster must be setup and passed as an argument if Dask is
+        used. Dask will need to be installed in the Python environment because it is not
+        a dependency of the project. More info can be found at 
         https://resqpy.readthedocs.io/en/latest/tutorial/multiprocessing.html
     """
     log.info("multiprocessing function called with %s function, %s entries.", function.__name__, len(kwargs_list))
 
-    tmp_dir = f'tmp_{uuid.uuid4()}'
+    tmp_dir = f'{tmp_dir_path}/tmp_{uuid.uuid4()}'
```

Comment:
Probably better to use os.sep instead of '/', or use os.path.join(), to construct the path.